src/libraries/Microsoft.PowerFx.Core/Public/Values/ImageValue.cs

```diff
@@ -0,0 +1,30 @@
+﻿// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+using System.Diagnostics.Contracts;
+using Microsoft.PowerFx.Core.Functions;
+using Microsoft.PowerFx.Core.IR;
+
+namespace Microsoft.PowerFx.Types
+{
+    public class ImageValue : FileValue
```

Comment:
>ImageValue [](http://example.com/codeflow?start=17&length=10)

This PR adds blobs, images and media, but only has (extensive) tests for blobs. I'd rather not add those classes that are not fully tested, adding them later and making the PR smaller (and easier to review :-)

Reply:
For tomorrow I can reduce the scope to Blob only, no connector update, no Image/Media
Then another PR for Media/Image
Then another for connectors